src/app/views/App.tsx

```diff
@@ -8,6 +8,9 @@ import { connect } from 'react-redux';
 import { bindActionCreators, Dispatch } from 'redux';
 
 import { geLocale } from '../../appLocale';
+import { telemetry } from '../../telemetry';
+import { SIDEBAR_HAMBURGER_BUTTON } from '../../telemetry/component-names';
+import { BUTTON_CLICK_EVENT } from '../../telemetry/event-types';
```

Comment:
Just a question, No action required. 
Can the constants be exported together so that we only have one import statement related to telemetry?

Reply:
It will only mean that we have to introduce the idea of namespaces in our code